### PR TITLE
OSDOCS-7204: Reorganize  CPMS info

### PR DESCRIPTION
**Pull request by jeana-redhat**
Version(s):
4.12+

Issue:
[OSDOCS-7204](https://issues.redhat.com//browse/OSDOCS-7204) Via [feedback in Slack](https://redhat-internal.slack.com/archives/CBZHF4DHC/p1690913524193009?thread_ts=1690904445.601729&cid=CBZHF4DHC)

Link to docs preview:
[Replacing a control plane machine](https://63028--docspreview.netlify.app/openshift-enterprise/latest/machine_management/control_plane_machine_management/cpmso-using.html#cpmso-feat-replace_cpmso-using)

QE review:
- [x] QE has approved this change.

Additional information:
[List of changes in this comment](https://github.com/openshift/openshift-docs/pull/63028#pullrequestreview-1559065939)

### CODE REVIEW

**Comment by jeana-redhat** [comment]
Small docs enhancement based on feedback from Emilien.

@damdo PTAL when you have a minute
@sunzhaohua2 this also needs QE review, please

File: modules/cpmso-feat-replace.adoc [hunk 1]
```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * machine_management/control_plane_machine_management/cpmso-using.adoc
+
+:_content-type: PROCEDURE
+[id="cpmso-feat-replace_{context}"]
+= Replacing a control plane machine
```
Comment (damdo) [inline modules/cpmso-feat-replace.adoc:7]: The concept of getting a control plane machine replaced is also true for the `RollingUpdate` strategy.

If you do the following steps with `RollingUpdate` the Operator still creates the required replacement machine.

So we could add something along the lines that independenty from the update strategy the user can follow these steps to get a machine replaced.

File: modules/cpmso-feat-replace.adoc [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+You can use the `OnDelete` update strategy to replace a control plane machine manually.
+
+For clusters that are configured to use the `OnDelete` update strategy, the Operator creates a replacement control plane machine when you delete an existing machine. When the replacement control plane machine is ready, the etcd Operator allows the existing machine to be deleted. The replacement machine then joins the control plane.
+
+If multiple control plane machines are deleted, the Operator creates all of the required replacement machines simultaneously.
+
+[NOTE]
+====
+You can use the `OnDelete` update strategy to test changes to your control plane configuration. Manually replacing machines allows you to test changes to your configuration on a single machine before applying the changes more broadly.
+====
```
Comment (damdo) [inline modules/cpmso-feat-replace.adoc:18]: Following on from my comment above, as it is correctly pointed out, this keeps being true only for the `OnDelete` strategy 👍

**Review by damdo — COMMENTED** [review]
Steps LGTM, just one note about strategies.

File: machine_management/control_plane_machine_management/cpmso-using.adoc
Comment (jeana-redhat) [inline machine_management/control_plane_machine_management/cpmso-using.adoc:1]: * Moved replace procedure to the top and promoted it to level one (basically making it its own topic outside the context of updating the config)
* Made `RollingUpdate` and `OnDelete` conceptual modules more parallel

File: modules/cpmso-feat-auto-update.adoc
Comment (jeana-redhat) [inline modules/cpmso-feat-auto-update.adoc:1]: * Title change to better conform to topic type=CONCEPT
* Removed passive voice as much as possible
* Explicitly discuss etcd health

File: modules/cpmso-feat-config-update.adoc
Comment (jeana-redhat) [inline modules/cpmso-feat-config-update.adoc:1]: * Removed passive voice as much as possible

File: modules/cpmso-feat-ondelete-update.adoc
Comment (jeana-redhat) [inline modules/cpmso-feat-ondelete-update.adoc:1]: * Title change to better conform to topic type=CONCEPT
* Made more parallel to `RollingUpdate` version
* Removed passive voice as much as possible
* Explicitly discuss etcd health

File: modules/cpmso-feat-replace.adoc
Comment (jeana-redhat) [inline modules/cpmso-feat-replace.adoc:1]: * Made context and procedure more unified across update strategies
* Explained update strategy differences for the multiple machine replacement case, with explicit mention of etcd health

**Review by jeana-redhat — COMMENTED** [review]
Thanks @damdo! Here are some notes on what I did, lmk what you think.

**Review by damdo — COMMENTED** [review]
Looks great @jeana-redhat 
Thanks for addressing my questions.
/lgtm

**Comment by nalhadef** [comment]
No comments in this one. :-)

**Pull request merged**